File: 1313-decompress-run-length-encoded-list/1313-decompress-run-length-encoded-list.js
/**
 * @param {number[]} nums
 * @return {number[]}
 */
var decompressRLElist = function(nums) {
  const result = []
  for (let i = 0; i < nums.length / 2; i++) {
    const len = nums[i*2]
    const val = nums[i*2+1]
    for (let j = 0; j < nums[i*2]; j++) {
      result.push(val)
    }
  }
  return result
};

File: 1313-decompress-run-length-encoded-list/1313-decompress-run-length-encoded-list.go
func decompressRLElist(nums []int) []int {
  var ans []int
  for i := 0; i < len(nums); i += 2 {
    v := nums[i+1]
    for j := 0; j < nums[i]; j++ {
      ans = append(ans, v)
    }
  }
  return ans
}